content/docs/user-guide/gems/reference/ros2/ros2-gem-user-guide.md

```diff
@@ -0,0 +1,209 @@
+
+### Supported Platforms and versions
+
+The Gem is currently Linux-only and is being tested with ROS 2 Humble on Ubuntu 22.04 as well as ROS 2 Galactic with Ubuntu 20.04.
+
+It is intended to support any modern ROS 2 version, following these priorities:
+
+- The most recent LTS version (e.g. in June 2022, [ROS 2 Humble](https://docs.ros.org/en/humble/Installation.html)).
+- The most recent non-LTS version ([ROS 2 Galactic](https://docs.ros.org/en/galactic/Installation.html)).
+- The always-fresh [ROS 2 Rolling](https://docs.ros.org/en/rolling/Installation.html).
+- Older versions.
+
+Currently tested and validated versions / platforms will be detailed in the [project repository](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2).
+
+**Currently, the O3DE ROS 2 Gem is not available for Windows.**
+ 
+If you have multiple versions installed, make sure you [source](https://docs.ros.org/en/humble/Tutorials/Workspace/Creating-A-Workspace.html#source-the-overlay) the one you would like to use. You can check which version is sourced in your console by checking the value of `ROS_DISTRO` environment variable (`echo $ROS_DISTRO`).
+
+### ROS 2 Concepts
+
+Please refer to [ROS 2 Concepts documentation](https://docs.ros.org/en/humble/Concepts.html).
+
+### Structure and Communication
+
+The Gem creates a [ROS 2 node](https://docs.ros.org/en/humble/Tutorials/Understanding-ROS2-Nodes.html) which is directly a part of the ROS 2 ecosystem. As such, your simulation will not use any bridges to communicate and is subject to configuration through settings such as Environment Variables. It is truly a part of the ecosystem.
+
+Note that the simulation node is handled through `ROS2SystemComponent` - a singleton. However, you are free to create and use your own nodes if you need more than one.
+
+Typically, you will be creating publishers and subscriptions. This is done through [rclcpp API](https://docs.ros2.org/humble/api/rclcpp/classrclcpp_1_1Node.html). Example:
+
+```
+auto ros2Node = ROS2Interface::Get()->GetNode();
+AZStd::string fullTopic = ROS2Names::GetNamespacedName(GetNamespace(), m_MyTopic);
+m_myPublisher = ros2Node->create_publisher<sensor_msgs::msg::PointCloud2>(fullTopic.data(), QoS());
+```
+
+Note that QoS class is a simple wrapper to [`rclcpp::QoS`](https://docs.ros.org/en/humble/p/rclcpp/generated/classrclcpp_1_1QoS.html).
+
+### Frames
+
+`ROS2FrameComponent` is a representation of an interesting physical part of the robot. It handles spatio-temporal relationship between this part and other frames of reference. It also encapsulates namespaces, which help to distinguish between different robots and different parts of the robot, such in the case of multiple identical sensors on one robot.
+
+All Sensors and the Robot Control Component require `ROS2FrameComponent`.
+
+### Sensors
+
+Sensors are Components deriving from `ROS2SensorComponent`. They acquire data from the simulated environment and publish it to ROS 2 domain.
```

Comment:
Nit: You can lowercase "component", it shouldn't be capitalized here. However, you can keep it capitalized when referring to a component's class name and using in-style code. On the other hand, "Gem" is always capitalized. (There are more instances of this throughout the page.)

```suggestion
Sensors are components deriving from `ROS2SensorComponent`. They acquire data from the simulated environment and publish it to ROS 2 domain.
```

Reply:
Adjusted in whole PR.